Swiftfin/Components/AlphaPicker/AlphaPickerView.swift

```diff
@@ -0,0 +1,63 @@
+//
+// Swiftfin is subject to the terms of the Mozilla Public
+// License, v2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at https://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) 2023 Jellyfin & Jellyfin Contributors
+//
+
+import Defaults
+import SwiftUI
+
+struct AlphaPickerView: View {
+
+    @ObservedObject
+    var viewModel: FilterViewModel
+    private var onSelect: (FilterCoordinator.Parameters) -> Void
+
+    @ViewBuilder
+    private var alphaPickerBody: some View {
+        VStack(spacing: 0) {
+            ForEach(AlphaPickerCharacters.letters.characters, id: \.self) { nameStartsWith in
+                AlphaPickerButton(
+                    nameStartsWith: nameStartsWith,
+                    activated: viewModel.currentFilters.nameStartsWith.contains {
+                        $0.id == nameStartsWith
+                    }, viewModel: viewModel) 
+                    .onSelect {
+                        onSelect(.init(
+                            title: L10n.letter,
+                            viewModel: viewModel,
+                            filter: \.nameStartsWith,
+                            selectorType: .single
+                    ))
+                }
+            }
+        }
+        .frame(width: 20)
```

Comment:
Don't set the `frame` here if you're also setting the `frame` when using the view.